sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.5.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
+      <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
```

Comment:
Update the tag to use the unreleased dependency.

Reply:
Updated